backends/urls.py

```diff
@@ -0,0 +1,24 @@
+"""URLs module"""
+from django.conf.urls import url
+
+from social_django import views
+from social_django.urls import extra
+
+from backends.views import complete
+
+
+app_name = 'social'
```

Comment:
Is this needed? Could you just import the `app_name` directly from `social_django.urls`?